Integrations/integration-symantec-DLP-incident-reportintg-update.yml

```diff
@@ -0,0 +1,209 @@
+            last_run['lt_time']= datetime.datetime.strptime(last_run['lt_time'],'%d/%m/%Y %H:%M:%S.%f')
+
+        incidentids=get_incident_id(last_run['lt_time'])
+        incidents=[]
+        if len(incidentids['incidentId']) > 0:
+            for incident1 in incidentids['incidentId']:
+                inc=incidentDetail(str(incident1))
+                incidents.append({
+                    'name':"DLP Incident #"+str(incident1),
+                    'details':  inc[0]['incident']['violatedPolicyRule'][0]['ruleName']+ "RULE has been violated"
+                })
+                if inc[0]['incident']['incidentCreationDate'].isoformat() > last_run['lt_time'].isoformat():
+                    demisto.setLastRun({'lt_time':inc[0]['incident']['incidentCreationDate'].strftime("%d/%m/%Y %H:%M:%S.%f")})
+        demisto.incidents(incidents)
+        return(incidents)
+
+
+    def updateIncidents():
+        args = demisto.args()
+        inc_id = args['incidentId']
+        args.pop('incidentId')
+        if args['status'] == 'New':
+            args['status'] = 'incident.status.New'
+        return client.service.updateIncidents(updateBatch={'batchId':str(random.randint(11111,99999)),'incidentId':inc_id,'incidentAttributes':args})
+
+
+
+    def myconverter(o):
+        if isinstance(o, datetime.datetime):
+            return o.__str__()
+
+
+
+    if demisto.command() == 'fetch-incidents':
+        fetchincidents()
+    elif demisto.command() =='test-module':
+        try:
+            inc_list = get_incident_id(datetime.datetime.now())
+            demisto.results("ok")
+        except:
+            return_error("Invalid instance details")
+    elif demisto.command() == 'dlp-incedent-details':
+        inc_details = incidentDetail( demisto.args()['incidentId'])
+        res = helpers.serialize_object(inc_details[0])
+        demisto.results(json.loads(json.dumps(res,default = myconverter)))
+    elif demisto.command() == 'dlp-update-incident':
+        demisto.results(updateIncidents())
+
+  type: python
+  commands:
+  - name: dlp-incedent-details
+    arguments:
+    - name: incidentId
+      required: true
+      description: incident Id  from DLP console to fetch the data of
+    description: fetches DLP incident details
+  - name: dlp-update-incident
+    arguments:
+    - name: incidentId
+      required: true
+      description: incident Id  from DLP console to fetch the data of
+    - name: severity
+      auto: PREDEFINED
+      predefined:
+      - HIGH
+      - MEDIUM
+      - LOW
+      - INFO
+      description: changes the severity of the Incident
+    - name: status
+      auto: PREDEFINED
+      predefined:
+      - New
+      - Escalated
+      - Investigation
+      - Resolved
+      - Dissmissed
+      description: 'changes the status of the incident in DLP console NOTE:  status
+        attributes are defined in the DLP console. '
+    description: updates Symantec DLP incidents
+  dockerimage: ssudheerk99/zeep
```

Comment:
@ssudheerk99 for the docker image we need to manage the docker creation in our sister project: https://github.com/demisto/dockerfiles . Please see if you can open a pr there to add the proper docker files and python dependencies. If you need any help with the docker creation we will be happy to assist (you can open an issue on the dockerfiles project if you have questions).

Reply:
@glicht  created a pr for zeep python package at https://github.com/demisto/dockerfiles 
@anara123   will update the the docker image name in the integration once it gets published in doceker hub 

:-)

Reply:
@glicht as the pr got merged when will be the zeep docker image will be available in docker hub ?